core/templates/pages/exploration-player-page/services/learner-view-rating.service.ts

```diff
@@ -36,6 +39,11 @@ export class LearnerViewRatingService {
     LearnerViewRatingBackendApiService
   ) {}
 
+  // Updates the view rating value of this particlaur instance of the
+  // LearnerViewRatingService class by using the controller scope object.
+  /**
+   * @params successCallback - function that recieves value from controller
```

Comment:
The format we generally follow is `* @param {<type definition>} <param name> - <description>`. Can we follow this pattern here? Here's an example:

https://github.com/oppia/oppia/blob/4fdf3f73348487e9a1673ff16de6006a6c41c00d/core/templates/domain/platform_feature/feature-status-summary.model.ts#L56-L57

Reply:
Ditto below for other functions.